etc/lime-elements.api.md

```diff
@@ -322,6 +322,12 @@ export namespace Components {
         "required": boolean;
         "value": FileInfo;
     }
+    export interface LimelFileDropzone {

```

Comment:
I wonder if we will get declaration merging here and export the event handlers, because this interface is now defined twice in this file 🤔. Worth double checking

Reply:
That file is automatically generated by api-extractor. @adrianschmidt do you have any input to Lawrence concern?

Reply:
Ah nevermind, this is probably intended. I see the other components have this repeated declaration as well 😅